developer-docs-site/docs/concepts/coin-and-token/aptos-token-v2.md

```diff
@@ -0,0 +1,128 @@
+NFTs are typically defined using the following properties:
+
+- `name`: The name of the asset. It must be unique within a collection.
+- `description`: The description of the asset.
+- `uri`: A URL pointer to off-chain for more information about the asset. The asset could be media such as an image or video or more metadata.
+- `supply`: The total number of units of this NFT. Many NFTs have only a single supply, while those that have more than one are referred to as editions.
+
+Additionally, most NFTs are part of a collection or a set of NFTs with a common attribute, for example, a theme, creator,
+or minimally contract. Each collection has a similar set of attributes:
+
+- `name`: The name of the collection. The name must be unique within the creator's account.
+- `description`: The description of the collection.
+- `uri`: A URL pointer to off-chain for more information about the asset. The asset could be media such as an image or video or more metadata.
+- `supply`: The total number of NFTs in this collection. 
+- `maximum`: The maximum number of NFTs that this collection can have. If `maximum` is set to 0, then supply is untracked. 
+
+## Design principles
+
+The [Aptos token v2 standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/token.move) 
+was developed with the following as an improvement on the Aptos Token standard.  It has these ideas in mind:
+* **Flexibility** - Allowing creators the ability to be creative in the space and allow for novel usage of NFTs
+* **Composability** - Allowing creators the ability to compose multiple NFTs together, such that the final object is greater than the sum of its parts
+* **Scalability** - Allowing for even greater parallelism between tokens
+
+Additionally, it took feedback from the community to add support for:
+* Instant transfers
+* Soul bound tokens
+* and much more
+
+## How Token V2 differs from Token V1
+
+Token V2 uses Aptos [objects](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/object.move)
+rather than account resources traditionally used in Move.  This allows for storing data outside the account and adding
+flexibility in this way.  
+* Transfers are simply a reference update
+* Direct transfer is allowed without an opt in
+* NFTs can own other NFTs adding easy composability
+* Soul bound tokens are supported
+
+## Token data model
+
+TODO: Diagram of the data model and associated links
+
+## Aptos Token -> No code tokens
+
+The [Aptos Token](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/aptos_token.move)
+module provides a no code solution for most use cases of NFTs.  It provides these main features:
+* Base token and collection features
+* Creator definable mutability for tokens
+* Creator-based freezing of tokens
+* Standard object-based transfer and events
+* Metadata property type
+
+### Royalties
```

Comment:
Let's also mention that royalty can be on the collection as well for a single collection-wide royalty setting